src/adaptation/stepsize.jl

```diff
@@ -8,7 +8,7 @@ mutable struct DAState{T<:AbstractScalarOrVec{<:AbstractFloat}}
     H_bar::T
 end
 
-computeμ(ϵ::AbstractScalarOrVec{<:AbstractFloat}) = log.(10 * ϵ)
+computeμ(ϵ::AbstractFloat) = log(10 * ϵ)
```

Comment:
I'd suggest that we define a vectorised version along with your proposal: 

```suggestion
computeμ(ϵ::AbstractFloat) = log(10 * ϵ)
computeμ(ϵ::AbstractVector{<:AbstractFloat}) = map(computeμ, ϵ)
```

The motivation is to keep the vectorised interface unchanged - we could refactor this interface in a separate effort.